### PR TITLE
[4828] added support to install externals from a directory (4-2-stable)

### PR DESCRIPTION
**Pull request by jassigill2000**

### CODE REVIEW

**Comment by trel** [comment]
please update this with information about testing success, and readiness to merge

File: irods_consortium_continuous_integration_build_hook.py [hunk 1]
```diff
@@ -34,11 +33,22 @@ def install_building_dependencies():
         'irods-externals-json3.7.3-0',
         'irods-externals-libarchive3.3.2-1',
         'irods-externals-zeromq4-14.1.6-0'
-        ])
+        ]
+    if externals_directory is None and externals_directory == 'None':
```
Comment (korydraughn) [inline irods_consortium_continuous_integration_build_hook.py]: I'm confused. Does this ever result in being `true`?

Comment (jassigill2000) [reply irods_consortium_continuous_integration_build_hook.py]: thank you for catching it.

**Comment by jassigill2000** [comment]
we can merge this I have built irods on ubuntu 16, ubuntu 18 and centos 7 you can merge this.

**Pull request merged**